### ISSUE TITLE
Unable to execute Meshroom UI

### ISSUE THREAD

**Issue by JimBilsborough** [alicevision/meshroom#621]
I'm a newbie here with limited experience. I deployed Meshroom to an AWS instance (Deep Learning Base AMI (Amazon Linux) Version 19.1, that comes with NVidia CUDA, cuDNN, NCCL, GPU Drivers, Intel MKL-DNN, etc.), using a step-by-step guide in https://www.instructables.com/id/Run-Photogrammetry-in-the-Cloud/. I successfully created a model in command line mode, I know it was fully successful because I downloaded the output to my local Win 10 machine and opened it in my local instance of Meshroom UI.

That's all well and good, but I really want to open the UI on my AWS instance, and I don't know how! The following files are in the AWS Meshroom directory:
![image](https://user-images.githubusercontent.com/55008338/64476236-78797b80-d15a-11e9-93e8-1f8a7ad18d9c.png)

I'm accessing my cloud platform using puTTY on a Win 10 machine with X10 forwarding enabled.

Can anyone give me any guidance on how to open the Meshroom interactive UI?

Jim

**Comment by simogasp** [comment]
You are connecting to a remote server. You should check that the server supports remote X11 forwarding or have something like X2Go or other utility for remote access via gui. 
I never used this AWS system but since you don't have the meshroom_ui file in your system it is likely that you cannot access to the UI, you can only execute the program as batch (which is waht AWS services are for, usually)

**Comment by JimBilsborough** [comment]
Thanks simogasp, that thought crossed my mind too.  However, your reply got me searching for AWS support for X11 - and it does. I'd failed to install a terminal emulator locally so I downloaded Xming, made some changes to sshd_config on my AWS platform, and installed xterm. I can successfully launch trivial GUI things like xclock and xeyes, so I know that AWS supports the kind of thing I want to do, but I can't for the life of me figure out how to launch Meshroom as a GUI app. I deployed the standard 2019.2.0 install package, So as far as I can tell, AWS supports X11 forwarding and my Meshroon install includes GUI access. I'm pretty sure I'm not missing much, but the solution still eludes me!

Thanks too for adding a **type:question** label to this post - newbie mistake!
